### PR TITLE
RM-124: Add command to perform a healthcheck against airbyte server

### PR DESCRIPTION
**Pull request by naswierczek**
- adds new optional arugment that can be passed to mvrec that will query the healthcheck endpoint on the configured airbyte instance and return either true or false depending on if that endpoint returns a 200 status code
- Added new class to support argument
- Added unit tests for new class method

### CODE REVIEW

File: records_mover/records/airbyte/airbyte.py
Comment (naswierczek) [inline records_mover/records/airbyte/airbyte.py:29]: This can't be hard-coded to local-airbyte

Comment (Brunope) [reply records_mover/records/airbyte/airbyte.py:29]: what should it be? required argument on the CLI? i'm okay with it defaulting to this maybe

File: records_mover/records/cli.py
Comment (Brunope) [inline records_mover/records/cli.py:128]: can we instead let the exception propagate so it's visible to the user? it'll sys exit anyway

File: records_mover/records/airbyte/airbyte.py
Comment (naswierczek) [reply records_mover/records/airbyte/airbyte.py:29]: Required argument doesn't make a ton of sense so having just a single stable entry we define is enticing. In production, we'll probably only have a single stable hostname for the airbyte instance. We'll have a different instance to run integration tests against and that's about it.

**Comment by naswierczek** [comment]
@Brunope This is just step one of an overall larger project to integrate airbyte as the engine for doing stuff. Anything else you'd want to see in this PR before it merges?  I'm feeling like I should put in a basic feature flag where if you don't have an environment variable set, the new option doesn't even appear.

**Review by Brunope — APPROVED** [review]
sorry i meant to click approve like last week but i guess forgot - i think this is going in the right general direction, good to merge once all the checks are passing

**Pull request merged**
